packages/jest-snapshot/src/__tests__/matcher.test.ts

```diff
@@ -5,15 +5,19 @@
  * LICENSE file in the root directory of this source tree.
  *
  */
-'use strict';
 
-const {toMatchSnapshot} = require('../');
+const jestSnapshot = require('../');
+
+const {toMatchSnapshot} = jestSnapshot;
 
 it(`matcher returns matcher name, expected and actual values`, () => {
   const actual = 'a';
   const expected = 'b';
   const matcher = toMatchSnapshot.bind({
-    snapshotState: {match: (testName, received) => ({actual, expected})},
+    snapshotState: {
+      // @ts-ignore
+      match: (testName: string, received: any) => ({actual, expected}), // eslint-disable-line
```

Comment:
?

Reply:
I'm wondering about both the ts ignore and the eslint disable

Reply:
`testName` and `received` are not used that is why TS is complaining about it

Reply:
Add a `_` in front of the variable name, and both eslint and tsc will shut up about it 🙂

Reply:
fwiw I think you can just remove the arguments if you don't use them. Doesn't hurt to keep them, though